rest_framework/static/rest_framework/docs/js/api.js

```diff
@@ -36,23 +69,23 @@ $('form.api-interaction').submit(function(event) {
     const key = form.data("key");
     var params = {};
 
-    const formData = new FormData(form.get()[0]);
-    for (var [paramKey, paramValue] of formData.entries()) {
+    const entries = formEntries(form.get()[0]);
+    for (var [paramKey, paramValue] of entries) {
```

Comment:
Something like:

```javascript
for (var i = 0; i < entries.length; i++) {
  var entry = entries[i],
      paramKey = entry[0],
      paramValue = entry[1];
      ...
}
```

I think we should probably also drop `const` and `Array.from`.